README.md

```diff
@@ -127,13 +127,73 @@ First-Party Sets is a proposal to standardize a mechanism that solves this issue
 
 # Applications
 
-In support of the various browser privacy models, first-party sets only control when embedded content that would otherwise be considered third-party can access its own state. Examples:
+Various applications of First-Party Sets have been suggested by different parties. The list below is an effort to compile suggested applications. 
+
+## Initial Proposed Applications
+
+**Bounce/navigation-based tracking protections**\
+_Interest from: [Safari](https://github.com/privacycg/first-party-sets/issues/53#issuecomment-902396090), Chrome, Edge_
+
+To allow for first-party cross-site single-sign-on flows to continue working, where [navigation-based tracking mitigations](https://privacycg.github.io/nav-tracking-mitigations/) are applied by browsers. Sign-on typically involves exchange of authentication tokens or other user identifiers via URL parameters on top-level navigation and redirects; which may get classified as navigational tracking.
+
+**Relaxing full third-party cookie blocking for cookies with the SameParty attribute**\
+_Interest from: Chrome, Edge_
+
+To help websites maintain various uses of cross-site cookies within the same organization such as for SSO, consent management, sandboxed domains, etc. 
+Cookies need to be labeled with the [SameParty attribute](#sameparty-cookies-and-first-party-sets).
+
+Prior art: Firefox and Edge use the [entity list](https://github.com/mozilla-services/shavar-prod-lists#entity-list), Safari’s [Quirks.cpp](https://webkit-search.igalia.com/webkit/rev/0be50a7e5c29546e4c5f27ee6b808f9aaadb9f80/Source/WebCore/page/Quirks.cpp#1154) and [affiliated domains](https://www.w3.org/2017/11/06-webappsec-minutes.html#item12) proposal.
```

Comment:
Thanks for the clarifications. How does this edit look to you?

```suggestion
Prior art: Edge uses the [entity list](https://github.com/mozilla-services/shavar-prod-lists#entity-list), Safari’s [Quirks.cpp](https://webkit-search.igalia.com/webkit/rev/0be50a7e5c29546e4c5f27ee6b808f9aaadb9f80/Source/WebCore/page/Quirks.cpp#1154) and [affiliated domains](https://www.w3.org/2017/11/06-webappsec-minutes.html#item12) proposal.

Similar idea used elsewhere: In their default tracking protection mode, Firefox currently also uses the Disconnect [entity list](https://github.com/mozilla-services/shavar-prod-lists#entity-list). However, Firefox intends for the usage of the entity list to be temporary, and plans to phase out its usage in the medium-to-long term. A combination of curated lists and heuristics may be used to treat any resulting site compatibility issues in the interim.
```

Reply:
@annevk @martinthomson Sorry, I just realized this forked our comment thread. Would you be able to take a look at my proposed edit and let us know if this looks okay to you? We'd like to merge in this PR as soon as resolved this thread.

Reply:
I don't see how it's a similar idea. This list is to actively block bad actors. Our plan is partitioning for everyone which to some extent makes blocking bad actors unneeded (but it might still be good for certain other attack vectors). FPS goes completely counter to our long term goal of having all authority derive from the address bar so I don't see how it's comparable.

Reply:
> I don't see how it's a similar idea. This list is to actively block bad actors.

The entity list? How so? :)

Reply:
`disconnect-entitylist.json` limits the impact of `disconnect-blacklist.json` to some extent, but there are no plans to make it undo partitioning or anything of that kind. I continue to fail to see the relevance.

Reply:
It's a list that declares domains owned by the same entity for purposes of avoiding breakage, shipped by a major browser. That seems relevant to me.

Reply:
It limits the impact of a blocklist which has the aim of limiting known trackers, but it doesn't relate to state partitioning or limiting the impact of state partitioning. Which is a separate feature we plan on shipping. So how it's relevant to FPS, which is about limiting the impact of state partitioning, is rather unclear to us.

Reply:
ETP targets 3rd party cookies as well, using a worst-actor blocklist and an entity list. The entity list itself bears a lot of resemblance to the list that is used in FPS, and sparked some inspiration for FPS. I don't see how this is in any way inaccurate. The text doesn't say that ETP is equal to FPS and/or state partitioning.

I think we can agree on the baseline truth that in the past, browsers have attempted to mitigate breakage from anti-tracking restrictions through the grouping of domains by common owner (and some of these attempts are still being shipped today). This is all this text is meant to convey, which is why it carefully tries to ensure that it's not misread as support from Mozilla.

And just to clarify, as this PR is intended to show more clearly, FPS has use cases outside of limiting the impact of 3rd party cookie restrictions like state partitioning.

Reply:
If you prefer, we could alter the beginning of that text to `In ETP, a default-enabled feature that selectively blocks 3rd party cookies from trackers, Firefox currently uses the entity-list...`? That may also give more context to the reader and make it clear why the entity list is mentioned in the section for 3rd party cookie blocking.